CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/base/BaseFragment.kt

```diff
@@ -48,9 +47,4 @@ abstract class BaseFragment : Fragment(), BaseFragmentCallbacks {
             bindViewModel()
         }
     }
-
-    open fun displayError(error: Throwable) {
-        val message = error.userReadableMessage(requireContext())
-        toaster.display(message)
-    }
```

Comment:
why do we remove this? I think It's useful to display default error messages on children fragments?